lib/v8debugapi.js

```diff
@@ -384,10 +384,27 @@ module.exports.create = function(logger_, config_, fileStats_) {
   // }
 
   function findScripts(scriptPath) {
+    // Use repository relative mapping if present
+    if (config.appPathRelativeToRepository) {
+      var candidate = scriptPath.replace(config.appPathRelativeToRepository,
+        config.workingDirectory);
```

Comment:
This was a conscious decision, I was hoping to support the use case you originally discussed in the bug:

```javascript
Server/app.js -> /app/app.js
Server/foo/app.js-> /app/foo/app.js
Server/app/app.js -> /app/app/app.js
```
If they happen to deploy their server directory to /app we want to support that too.